app/views/layouts/application.html.erb

```diff
@@ -1,66 +1,64 @@
 <!DOCTYPE html>
 <html>
-<head>
-	<title>ProbableEngine</title>
-	<%= Gon::Base.render_data({}) %>
+  <head>
+    <title>ProbableEngine</title>
+    <%= Gon::Base.render_data({}) %>
 
-	<%= stylesheet_link_tag 'application', media: 'all' %>
-  <%= stylesheet_link_tag "#{params[:controller]}/#{params[:action]}" if File.exist?("#{Rails.root}/app/assets/stylesheets/#{params[:controller]}/#{params[:action]}.css") %>
-  <%= javascript_include_tag 'application' %>
-  <%= javascript_include_tag "#{params[:controller]}/#{params[:action]}" if File.exist?("#{Rails.root}/app/assets/javascripts/#{params[:controller]}/#{params[:action]}.js") %>
```

Comment:
we're precompiling assets, we dont need to be including all assets and specific assets.